src/ILVerification/src/ILVerification.csproj

```diff
@@ -1,12 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <CLSCompliant>false</CLSCompliant>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <TargetFramework>netstandard1.5</TargetFramework>
+    <AssemblyOriginatorKeyFile>src\BuildIntegration\StrongNameKeys\ILVerification.snk</AssemblyOriginatorKeyFile>
```

Comment:
BuildIntegration directory is meant to contain files that are part of the ILCompiler nuget package. Could you please put them these under src\ILVerification instead?